rvt-patch-llvm/src/main.rs

```diff
@@ -392,12 +392,36 @@ where
     None
 }
 
-fn handle_panic(module: &Module) {
+fn handle_panic(context: &Context, module: &Module) {
     // TODO: make "spanic" a CL arg.
     if let Some(spanic) = module.get_function("spanic") {
-        if let Some(unwind) = module.get_function("rust_begin_unwind") {
-            unwind.replace_all_uses_with(spanic);
-            info!("Replaced panic handling ('rust_begin_unwind') with 'spanic'.");
+        // Collect all the functions that start with
+        // "_ZN3std9panicking11begin_panic17h" (there will be more than one).
+        let mut panics = vec![];
+        let mut op_fun = module.get_first_function();
+        while let Some(fun) = op_fun {
```

Comment:
Also... I don't understand why there is more than one - what is the difference between them?
And how can there be more than one - do the different versions found differ only in the hash at the end?

Reply:
I changed the get_function_* functions to return Vec instead of Option.

> Also... I don't understand why there is more than one - what is the difference between them?

I don't understand either. I see multiple instances with different hash suffix. They all have the same body.